app/views/thesis/new.html.erb

```diff
@@ -31,7 +31,7 @@
       <div class='group-inline'>
 
       <%= f.input :graduation_month, label: 'Month', as: :select,
-                           collection: Date::MONTHNAMES.compact,
+                           collection: Thesis::VALID_MONTHS,
```

Comment:
Nice. I hadn't looked at how that was originally coded and wasn't sure if this was gonna bit a pain or not. 👍